Makefile

```diff
@@ -316,10 +336,12 @@ release: $(GORELEASER) #EXHELP Runs goreleaser for the operator-controller. By d
 	$(GORELEASER) $(GORELEASER_ARGS)
 
 .PHONY: quickstart
-quickstart: export MANIFEST := https://github.com/operator-framework/operator-controller/releases/download/$(VERSION)/operator-controller.yaml
+quickstart: export MANIFEST := ./operator-controller.yaml
 quickstart: $(KUSTOMIZE) manifests #EXHELP Generate the installation release manifests and scripts.
 	$(KUSTOMIZE) build $(KUSTOMIZE_BUILD_DIR) | sed "s/:devel/:$(VERSION)/g" > operator-controller.yaml
-	envsubst '$$CATALOGD_VERSION,$$CERT_MGR_VERSION,$$INSTALL_DEFAULT_CATALOGS,$$MANIFEST' < scripts/install.tpl.sh > install.sh
+	cd catalogd && $(KUSTOMIZE) build config/overlays/cert-manager > catalogd.yaml
```

Comment:
Similar comment that I made in `kind-deploy`. Also I assume we'll need a similar sed command for catalogd's manifest?

Reply:
Done but still need to do more changes.